packages/react-chat/examples/index.html

```diff
@@ -32,7 +32,6 @@
           },
           render: {
             mode: 'overlay',
-            target: document.querySelector('#flat-chat'),
```

Comment:
not sure why we would have a target with `'overlay'`

Reply:
for testing only. thanks 👍